app/screens/receive-bitcoin-screen/receive-screen.tsx

```diff
@@ -265,6 +273,15 @@ const ReceiveScreen = () => {
           convertMoneyAmount={request.convertMoneyAmount}
           walletCurrency={request.receivingWalletDescriptor.currency}
           showValuesIfDisabled={false}
+          minAmount={
+            request.defaultWalletDescriptor.currency === "BTC" && isFirstTransaction
```

Comment:
remember, the default wallet currency doesn't have to be BTC, it could be USD, and they are just selecting the BTC wallet to use it, so you maybe want to check the receivingWalletDescriptor

Reply:
Also, change the text from "initial deposit of (~$1) required..." to  "initial deposit of (~$5) required..."

Reply:
and the reverse is true, when BTC is the default wallet, and I switch to USD tab, it still shows "initial deposit of $1 required.." that message should disappear when USD tab is selected. 

So the test cases are:

1) Default Wallet = BTC, generate BTC invoice
2) Default Wallet = BTC, generate USD invoice
3) Default Wallet = USD, generate USD invoice
4) Default Wallet = USD, generate BTC invoice